frontend/src/routes/create-goals/+page.svelte

```diff
@@ -95,211 +80,379 @@
+    <!-- Time-Bound card -->
+    <div class="card w-full">
+      <figure>
+        <figcaption class="p-4 card-body flex flex-col">
+          <h2 class="card-title mb-2">Time-Bound</h2>
+          <div class="flex flex-col md:flex-row w-full">
             <input
               id="time-bound"
-              class="shadow appearance-none border rounded w-full py-2 px-3 leading-tight focus:outline-none focus:shadow-outline"
+              class="shadow appearance-none border rounded-xl w-full py-2 px-3 leading-tight focus:outline-none focus:shadow-outline flex-grow mb-2 md:mb-0"
               type="text"
               placeholder="AchievAIm's Time-Bound suggestion"
               bind:value={goal.time_bound}
             />
-            <div class="mt-3">
-              <button class="btn btn-primary">Keep Time-Bound Suggestion</button>
-            </div>
+            <label
+              class="cursor-pointer label flex items-center md:ml-2 mt-2 md:mt-0 w-full md:w-auto justify-end"
+            >
+              <input type="checkbox" class="toggle toggle-primary" checked />
+              <div class="dropdown dropdown-end">
+                <button tabindex="0" class="btn btn-circle btn-ghost btn-xs text-info m-3">
+                  <svg
+                    xmlns="http://www.w3.org/2000/svg"
+                    fill="none"
+                    viewBox="0 0 24 24"
+                    class="w-4 h-4 stroke-current"
+                  >
+                    <path
+                      stroke-linecap="round"
+                      stroke-linejoin="round"
+                      stroke-width="2"
+                      d="M13 16h-1v-4h-1m1-4h.01M21 12a9 9 0 11-18 0 9 9 0 0118 0z"
+                    />
+                  </svg>
+                  <button class="card compact dropdown-content z-[1] bg-base-100 rounded-box w-64">
+                    <div class="card-body">
+                      <h2 class="card-title">You needed more info?</h2>
+                      <p>Here is a description!</p>
+                    </div>
+                  </button>
+                </button>
+              </div>
+            </label>
+          </div>
+        </figcaption>
+      </figure>
+    </div>
+
+    <div class="mt-3 flex flex-col items-center">
+      <div class="card w-full">
+        <figure>
+          <figcaption class="p-4 card-body flex flex-col items-center">
+            {#if goal.days_of_week}
+              <div class="flex justify-between items-center w-full">
+                <h2 class="card-title mb-2">Days</h2>
+
+                <div class="flex items-center">
+                  <label for="selectAll" class="cursor-pointer label flex items-center">
+                    <input
+                      type="checkbox"
+                      class="toggle toggle-primary"
+                      id="selectAll"
+                      bind:checked={selectAll}
+                      on:click={toggleAll}
+                    />
```

Comment:
I think we should put some kind of label next to this explaining what it is for. When I saw it on the screen I thought it was to lock the values. I didn't realize the real purpose until I looked at the code.

Reply:
In the most reccent pull request I updated the i, icon for information about the selector.